### ISSUE TITLE
TextField Keyboard dosent  show or work when i focous inside it .

### ISSUE THREAD

**Issue by sabahya3** [X-SLAYER/flutter_overlay_window#1]
I try to show a box over apps and inside of it I added a TextField , and when i press inside of it to write it dosent show the keyboard to enable me to write inside the TextField

**Comment by X-SLAYER** [comment]
I have just published a new version that fixes the bug [flutter_overlay_window v0.0.2](https://pub.dev/packages/flutter_overlay_window/versions/0.0.2)
use the `OverlayFlag.flagNotTouchModal` flag when u use text fields inside the overlay 
like that
```dart
await FlutterOverlayWindow.showOverlay(flag: OverlayFlag.flagNotTouchModal);
```

**Comment by sabahya3** [comment]
Thanks bro,  I will try it and give you feed back a lot of love from Egypt
💙💙
